Cargo.toml

```diff
@@ -58,6 +58,7 @@ dashmap = "3.11.10"
 string_cache = "0.8.0"
 postgres-types = { version = "0.2", features = ["derive"] }
 getrandom = "0.2.1"
+funty = "=1.1.0"
```

Comment:
What is this new crate for?

Reply:
Not needed anymore apparently. iirc, it was creating a dependency conflict.